setup.sh

```diff
@@ -72,16 +72,8 @@ install_opencv() {
   pip3 install opencv-python
 }
 
-install_keras() {
-  pip3 install keras
-}
-
-install_tensorflow() {
-  pip3 install tensorflow
-}
-
-install_h5py() {
```

Comment:
These removed functions are still called at the bottom of the script (lines 96 to 99). Could you update them to call the new function?